src/site/markdown/server/manage-webadmin.md

```diff
@@ -1545,6 +1545,45 @@ Response codes:
  - 400: The `fromDomain` resource name is invalid
  - 400: The destination domain specified in the body is invalid
 
+## Creating address regex
+You can use **webadmin** to create regex mappings.
+
+When a request containing a JSON body with two fields `source` and `regex` is sent, the route will create the regex mapping.
```

Comment:
This is too much of a detail. Here explain me what a regex mapping is and does.

Reply:
It was an old version of addRegexMapping documentation, I got many comments for it, and I updated the content. I removed it so we could focus on removeRegexMapping documention.

Reply:
Not sure if the description can be shorter. I see it is short and descriptive enough.